src/github/githubRepository.ts

```diff
@@ -203,103 +203,44 @@ export class GitHubRepository implements IGitHubRepository, vscode.Disposable {
 		};
 	}
 
-	async getPullRequests(prType: PRType, page?: number): Promise<PullRequestData | undefined> {
-		return prType === PRType.All ? this.getAllPullRequests(page) : this.getPullRequestsForCategory(prType, page);
-	}
+	async getPullRequestsGraphQL(type: PRType, nextCursor?: string|null):Promise<PullRequestListResponse|undefined> {
+		const { remote, query, octokit } = await this.ensure();
+		const currentUser = octokit && (octokit as any).currentUser;
+		const currentUserLogin: string = currentUser.login;
 
-	private async getAllPullRequests(page?: number): Promise<PullRequestData | undefined> {

```

Comment:
I don't think we should remove this - we still need to support GHE that doesn't have GraphQL